apis/core/v1alpha1/conditions.go

```diff
@@ -41,6 +41,10 @@ const (
 	//		- ServiceUnavailable errors that are transient
 	//		- AccessDeniedException that needs correct credentials
 	ConditionTypeRecoverable ConditionType = "ACK.Recoverable"
+	// ConditionTypeWarning indicates any warning that may be present in the resource.
+	// Examples include
+	//      - Modifying an immutable field after it was created
+	ConditionTypeWarning ConditionType = "ACK.Warning"
```

Comment:
I actually would prefer to call this `ConditionTypeInfo` or `ConditionTypeAdvisory` instead of `ConditionTypeWarning`. The word "warning" is generally reserved for serious things that should be addressed ASAP by an operator. In the case of an immutable field being modified in the `Spec` of a resource, that's definitely not something that needs to be addressed ASAP by an operator (since it's not dangerous or anything)

Reply:
Changed this to `ConditionTypeAdvisory` as someone may ignore `ConditionTypeInfo` without noticing